Disable the tests, cuts build time by half and avoids build issues
with extra dependencies.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>

diff -Nura libmemcached-1.0.18.orig/Makefile.am libmemcached-1.0.18/Makefile.am
--- libmemcached-1.0.18.orig/Makefile.am	2014-07-19 09:24:45.246418899 -0300
+++ libmemcached-1.0.18/Makefile.am	2014-07-19 09:25:12.701349345 -0300
@@ -1,6 +1,6 @@
 # vim:ft=automake
 
-ACLOCAL_AMFLAGS= -I m4 -I libtest/m4
+ACLOCAL_AMFLAGS= -I m4
 AM_YFLAGS= -d
 
 # includes append to these:
@@ -45,8 +45,6 @@
 include libmemcached-1.0/include.am
 else
 
-include libtest/include.am
-
 include libhashkit-1.0/include.am
 include libmemcached-1.0/include.am
 
@@ -76,7 +74,6 @@
 include libmemcachedinternal/util/include.am
 include rpm/include.mk
 include support/include.am
-include tests/include.am
 include util/include.am
 include win32/include.am
 
@@ -103,7 +100,7 @@
 	@echo "Files that need to be either removed or checked in:"
 	@bzr unknowns
 
-clean-local: clean-libtest-check clean-docs-check clean-rpm
+clean-local: clean-docs-check clean-rpm
 
 
 lcov: lcov-clean check
